boards/xtensa/esp32/esp32-wrover-kit/configs/autopm/defconfig

```diff
@@ -28,8 +28,8 @@ CONFIG_ESP32_SPIFLASH=y
 CONFIG_ESP32_SPIFLASH_SPIFFS=y
 CONFIG_ESP32_STORAGE_MTD_SIZE=0x80000
 CONFIG_ESP32_UART0=y
+CONFIG_ESP32_WIFI=y
 CONFIG_ESP32_WIFI_SAVE_PARAM=y
-CONFIG_ESP32_WIRELESS=y
```

Comment:
is this a "breaking change"? I mean do users need to change `CONFIG_ESP32_WIRELESS=y` -> `CONFIG_ESP32_WIFI=y` in their configs to get the same functionality as before?